editor/fileserver/editor_file_server.cpp

```diff
@@ -258,7 +258,7 @@ void EditorFileServer::_subthread_start(void *s) {
 			} break;
 			case FileAccessNetwork::COMMAND_CLOSE: {
 
-				print_line("CLOSED");
+				print_line(TTR("CLOSED"));
```

Comment:
Debugging strings don't need to be translated – they should be removed instead. This should probably be done in a separate pull request.